src/main/java/in/ac/bits/protocolanalyzer/persistence/repository/ElasticsearchConfig.java

```diff
@@ -49,7 +53,19 @@ public ElasticsearchOperations elasticsearchTemplate() {
                 .put("http.cors.allow-methods", allowMethods)
                 .put("http.cors.allow-headers", allowHeaders);
         NodeBuilder builder = NodeBuilder.nodeBuilder().local(true)
-                .settings(settingsBuilder.build());
+                .settings(settingsBuilder.build());*/
+	ImmutableSettings.Builder settingsBuilder = esFactoryImpl.settingsBuilder().put("cluster.name", clusterName)
```

Comment:
All of these settings can be pushed to the ESFactoryImpl.settingsBuilder() method. An invocation of that method exists here to represent the idea of creating the setting builder. This chain invocation disappears into the ESFactoryImpl.settingsBuilder() method.